src/Symfony/Bridge/Doctrine/PropertyInfo/DoctrineExtractor.php

```diff
@@ -27,11 +28,22 @@
  */
 class DoctrineExtractor implements PropertyListExtractorInterface, PropertyTypeExtractorInterface
 {
+    private $entityManager;
     private $classMetadataFactory;
```

Comment:
I don't think so, it's a private property anyway

Reply:
It might help at time to clean up deprecated features from the next major branch. 
With some well placed `@deprecated` flags for each deprecated feature, cleaning an entire component can be way easier (just `grep`ing) and free of leftovers (each one leading to "drop dead code" PRs, you know).

Reply:
that, and the fact it hints to not use the property anymore